tornado/options.py

```diff
@@ -41,6 +41,13 @@ def connect():
     # or
     tornado.options.parse_config_file("/etc/server.conf")
 
+.. note:
+
+   When using tornado.options.parse_config_file, the only options that are set
+   are ones that were previously defined with either tornado.options.parse_command_line
```

Comment:
`parse_command_line` doesn't define any options; the only way to define options is with the `define` function.